release/models/bgp/openconfig-bgp-common-multiprotocol.yang

```diff
@@ -367,10 +374,10 @@ submodule openconfig-bgp-common-multiprotocol {
     description
       "Grouping for configuration common to all AFI,SAFI";
 
-    container prefix-limit {
+    container prefix-limit-accepted {
```

Comment:
My preference is to leave this container as-is.  But keeping the clarification in the description is a good idea

Reply:
Address via: https://github.com/openconfig/public/pull/532/commits/d53f9c098ecf4c9cf34c82233294e3bc30048991